Python/wrfrun/unCMIP6.py

```diff
@@ -437,7 +499,7 @@ def interp_data(self):
                     Temp = ds.values
                     Temp = Temp+273.15
                     ds.values = Temp
-                # NOTE: This is because in MPI-ESM1-2-HR data tos is in degress C and WRF expects degrees K.
+                # NOTE: This is because in some model data tos is in degress C and WRF expects degrees K.
```

Comment:
Why all these temporary variables? How about `ds += 273.15`? or at least `ds = ds + 273.15`?

Reply:
I removed the Temp variables as much as I could. I replaced the ds with da where applicable.